test/test.c

```diff
@@ -165,7 +165,7 @@ static int path_is_dir(const char *path)
 {
 	struct stat astats;
 
-	if ((lstat(path, &astats)) != 0)
+	if ((stat(path, &astats)) != 0)
```

Comment:
Good catch. `path_is_file()` too needs to be fixed, to use `stat` instead of `lstat`

Reply:
I have pushed a commit that fixes path_is_file()